src/elements/common/sub-header/SubHeaderRight.tsx

```diff
@@ -97,9 +102,16 @@ const SubHeaderRight = ({
             )}
 
             {isMetadataView && isMetadataViewV2Feature && hasSelectedItems && (
-                <Button icon={Pencil} size="large" variant="primary" onClick={onMetadataSidePanelToggle}>
-                    {formatMessage(messages.metadata)}
-                </Button>
+                <>
+                    {(selectedItemIds === 'all' || (selectedItemIds instanceof Set && selectedItemIds.size > 0)) &&
+                        bulkItemActions &&
+                        bulkItemActions.length > 0 && (
+                            <BulkItemActionMenu actions={bulkItemActions} selectedItemIds={selectedItemIds} />
+                        )}
```

Comment:
we need a unit tests to make sure there isnt a regression here

Reply:
Added unit tests in subheader right for rendering (ensuring elements exist with correct props), that components don't render with incorrect propps, and that onClick is called.